app/src/main/java/com/github/multimatum_team/multimatum/activity/DisplayLocationActivity.kt

```diff
@@ -0,0 +1,108 @@
+package com.github.multimatum_team.multimatum.activity
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import androidx.annotation.DrawableRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.content.res.AppCompatResources
+import com.github.multimatum_team.multimatum.R
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.MapView
+import com.mapbox.maps.Style
+import com.mapbox.maps.plugin.annotation.annotations
+import com.mapbox.maps.plugin.annotation.generated.PointAnnotationOptions
+import com.mapbox.maps.plugin.annotation.generated.createPointAnnotationManager
+
+class DisplayLocationActivity : AppCompatActivity() {
+
+    private var mapView: MapView? = null
+    private var latitude: Double = 0.0
+    private var longitude: Double = 0.0
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_display_location)
+
+        // Initializing the map
+        initializeMapView()
+    }
+
+    private fun initializeMapView() {
+        mapView = findViewById(R.id.mapView)
+        mapView?.getMapboxMap()?.loadStyleUri(
+            Style.MAPBOX_STREETS
+        ) { addAnnotationToMap() }
+
+        // Getting coordinates from the intent
+        val intent = intent
+        latitude = intent.getDoubleExtra("latitude", 0.0)
+        longitude = intent.getDoubleExtra("longitude", 0.0)
+
+        mapView?.getMapboxMap()?.also {
+            it.setCamera(
+                CameraOptions.Builder()
+                    .center(Point.fromLngLat(longitude, latitude))
+                    .zoom(DEFAULT_ZOOM)
+                    .build()
+            )
+        }
+    }
```

Comment:
Again minor, but a bit more readable in my opinion:
```suggestion
        mapView?.getMapboxMap()?.apply {
            setCamera(
                CameraOptions.Builder()
                    .center(Point.fromLngLat(longitude, latitude))
                    .zoom(DEFAULT_ZOOM)
                    .build()
            )
        }
    }
```

Reply:
Yes, I agree